src/main/perl/CacheManager/Element.pm

```diff
@@ -640,61 +580,34 @@ SWITCH:
     return $ret;
 }
 
+
 #
 # _resolve_eid($prof_dir, $ele_path)
 #
-# Private function that resolve element's id number. $prof_dir is the profile
+# Private function that resolve element's encoded id number. $prof_dir is the profile
 # full directory path, and $ele_path is the element path (as string)
 #
-sub _resolve_eid
+sub _resolve_enc_eid
```

Comment:
can you reread the pod in `Encode.pm` to see if it makes any sense at all? (it should also explain what an `eid` is (and it's not "encoded id", it's probably short for `euhm, index?` which is the response to "and how should we abbreviate this path counter?" 😄 )